src/test/java/com/telefonica/iot/cygnus/backends/ckan/CKANBackendImplTest.java

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
I saw many more cases so I leave it to your discretion to act or not upon it... :)